service/dap/types.go

```diff
@@ -242,6 +242,15 @@ type AttachConfig struct {
 	// The numeric ID of the process to be debugged. Required and must not be 0.
 	ProcessID int `json:"processId,omitempty"`
 
+	// Wait for a process with a name beginning with this prefix.
+	AttachWaitFor string `json:"waitFor,omitempty"`
+
+	// Interval between checks of the process list, in millisecond.
```

Comment:
1. Is it possible to come up with good default values for AttachWaitForInterval and AttachWaitForDuration, and not exposing them in the args yet? I am not sure about the usefulness of `AttachWaitForDuration` param in IDE UX yet, and I don't know when and why users are supposed to tweak AttachWaitForInterval. How about waiting until we find they are needed?

2. If we need them, can we use time.Duration instead of float64? And also comment what are the default values.
(I see these are from waitfor-interval and waitfor-duration flags, but wonder why they weren't using flag.Duration and got converted internally)

Reply:
I agree with your point of view, not to expose these attributes